src/Gui/Viewer/WindowQt.inl

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include <Gui/Viewer/WindowQt.hpp>
+
+namespace Ra {
+namespace Gui {
+
+class WindowQt::ScopedContext
```

Comment:
Question about the naming: is this necessary related to GL ? If yes, it could be named `ScopedGLContext`.

Reply:
Yes it's related to GL. Do you suggest to rename the inner class only or also the activateScopedContext method ?
Other Qt/GL related fonction are named context, makeCurrent and doneCurrent, following the Qt's convention. 
So my first opinion is ok to rename inner class, keep method name

Reply:
Ok, sounds good, I guess this will be more readable.